mixer/adapter/metadata/inventory.go

```diff
@@ -0,0 +1,360 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package metadata contains all compiled-in adapter metadata
+package metadata
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gogo/googleapis/google/rpc"
+	"github.com/gogo/protobuf/types"
+
+	bypass "istio.io/istio/mixer/adapter/bypass/config"
```

Comment:
Correct me if I am wrong, but this really doesn't break the build dependency, right?

Reply:
I think it's acceptable to have a dependency on protobuf packages. These config packages only have protobufs with no link to the 3rd party library implementing the adapter.